File: src/main/java/com/pei/dehaze/model/query/RolePageQuery.java
package com.pei.dehaze.model.query;

import com.pei.dehaze.common.base.BasePageQuery;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * 角色分页查询实体
 *
 * @author earthyzinc
 * @since 2022/6/3
 *
 */
@Data
@EqualsAndHashCode(callSuper = true)
public class RolePageQuery extends BasePageQuery {

    @Schema(description="关键字(角色名称/角色编码)")
    private String keywords;
}

File: dehaze-go/internal/model/query/role_page_query.go
package query

// RolePageQuery 角色分页查询对象
type RolePageQuery struct {
	// 关键字(角色名称/角色编码)
	Keywords string `json:"keywords"`
	// 页码
	PageNum int `json:"pageNum"`
	// 每页条数
	PageSize int `json:"pageSize"`
}
